Rubberduck.Core/UI/CodeExplorer/Commands/AddComponentCommand.cs

```diff
@@ -51,6 +53,60 @@ public void AddComponent(CodeExplorerItemViewModel node, ComponentType type)
             }
         }
 
+        public void AddComponent(CodeExplorerItemViewModel node, string moduleText)
+        {
+            var nodeProject = GetDeclaration(node)?.Project;
```

Comment:
Isn't `Project` here the `IVBProject`? If so, we are leaking it, no? Probably should be in a `using`?

Reply:
Wouldn't releasing the `IVBProject` cause problems for anything that later needs to access that specific `Declaration` node?

Reply:
You might be right... but that's still wrong for a different reason - why should `Declaration` class should be toting along a SCW here? We should be using QMN, or at least returning a brand new `IVBProject` to be then disposed by the caller.

Reply:
As little obvious as it may be here, this `IVBProject` is ultimately managed by the projects repository.

Reply:
I think this concretes the case for an App Hungarian naming convention to help track that.